backend/dal/Migrations/20201019200725_PA-635.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class PA635 : Migration
```

Comment:
Rename to `v01.01.00`.  If we ensure the first migration of a sprint has the final version name, it allows us to merge down to that one without forcing anyone to rebuild their DB.